### DISCUSSION TITLE
code is tried and tested on PHP Version 8.3.11

### DISCUSSION THREAD

**Discussion by sarangtc** [Larian77/Openbiblio#46] [Show and tell]
I fixed some deprecation issues, so we might even be OK to PHP9
I also removed my comments "// Changes PVD(8.0.x)" that I had put before all changes I made to make it php8.0 compatible as it was causing unnecessary clutter

**Comment by oppiet30** [comment]
I was going to try and update the 0.7.2 version to work with PHP 8.3.0+ But I don't know enough. Was stuck on how to handle Error messages.

**Reply by Larian77** [reply]
> I was going to try and update the 0.7.2 version to work with PHP 8.3.0+ But I don't know enough. Was stuck on how to handle Error messages.

The 0.7.x-versions doesn't work with PHP 0.8.x, you have to use Openbiblio 0.8.x from this repository! In this version there are also installation instructions. You can also download it from https://openbiblio.de/download.php
